src/lib.rs

```diff
@@ -47,7 +47,16 @@ mod arch {
         verbose: bool,
         ignore_status: bool,
     ) -> Option<String> {
-        let perf = env::var("PERF").unwrap_or_else(|_| "perf".to_string());
+        let perf = if let Ok(path) = env::var("PERF") {
+            path
+        } else {
+            if Command::new("perf").arg("--help").status().is_err() {
```

Comment:
Suppressing the output from this: https://github.com/flamegraph-rs/flamegraph/pull/355